src/output/plugins/visualization/VisualizationClient.cxx

```diff
@@ -0,0 +1,533 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The Music Player Daemon Project
+
+#include "VisualizationClient.hxx"
+
+#include "Log.hxx"
+#include "event/Chrono.hxx"
+#include "lib/fmt/ThreadIdFormatter.hxx"
+#include "util/Domain.hxx"
+
+#include <chrono>
+#include <thread>
+#include <variant>
+
+const Domain d_vis_client("vis_client");
+
+inline
+typename std::chrono::microseconds::rep
+NowTicks() {
```

Comment:
This function is only used for your excessive debug prints, but those should be removed before I merge this. What's even the point of this function, why do you insert those time stamps in all log messages?